### DISCUSSION TITLE
Subscribing to another calendar's iCal feed?

### DISCUSSION THREAD

**Discussion by curtzt** [liedekef/events-made-easy#689] [Q&A]
I have an Events Made Easy calendar; I 'd like to subscribe to another Events Made Easy calendar, on a different website, so that events entered in that calendar showed up in my calendar. Is there some way to do that using the ical feed from the calendar on the other website, or through some other mechanism?

**Comment by liedekef** [comment]
sorry, but EME is not meant for that. You can use any calendar app that can subscribe to multiple feeds to achieve this.

**Comment by curtzt** [comment]
Thanks for the extremely prompt reply! (I just wanted to avoid having to add the same events manually to each of the two website’s calendars, and - less importantly - to avoid any user who was interested having to subscribe to two ical feeds.)

Best wishes,
Thad Curtz

**Reply by liedekef** [reply]
Well, the calendar in EME is currently a very hackish thing but it gets the job done (and works with filters if needed). Finding a php lib that would allow the same functionality would cost me a lot of time (and coding), so currently best is to use aother wp plugin if you want to combine multiple feeds.

**Comment by curtzt** [comment]
Thanks! I really like EME, and I’m used to it, so I’ll solve this problem some other way…

Take care,
Thad
